packages/components/src/rating/stories/index.js

```diff
@@ -0,0 +1,12 @@
+/**
+ * Internal dependencies
+ */
+import Rating from '../';
+import '../style.scss';
+
+export default {
+	title: 'WooCommerce Admin/components/Rating',
+	component: Rating,
+};
+
+export const Default = () => <Rating rating={ 4.6 } totalStars={ 5 } />;
```

Comment:
Would be awesome to define some knobs in here to control `rating`, `totalStars`, and `size` to demo some Storybook functionality for upcoming components.

Reply:
Good point, I'll try that (a good excuse to play with knobs) :)

Reply:
Thanks for the nudge - added knobs for all three props 🎛